python/ray/_private/workers/default_worker.py

```diff
@@ -222,6 +231,10 @@
     )
     configure_log_file(out_file, err_file)
 
+    if mode == ray.WORKER_MODE and args.worker_preload_modules:
+        module_names_to_import = args.worker_preload_modules.split(",")
+        ray._private.utils.try_import_each_module(module_names_to_import)
```

Comment:
should be pretty straightforward to write a test using https://docs.python.org/3/library/sys.html#sys.modules?